test/ScanForErrorMiddleware/ScannerTest.php

```diff
@@ -125,10 +139,13 @@ public function testScanningEmitsInfoToConsoleWhenEncounteringFilesOfInterest()
         $this->console
             ->writeLine(
                 Argument::that(function ($arg) {
-                    if (! strstr($arg, 'src/ErrorMiddleware.php')) {
+                    if (false === strpos($arg, 'src/ErrorMiddleware.php')) {
                         return false;
                     }
-                    if (! strstr($arg, sprintf('<error>implementing %s</error>', ErrorMiddlewareInterface::class))) {
+                    if (false === strpos(
```

Comment:
On consideration, changing all occurrences of `! strstr` to `false === strpos` would make it more internally consistent, so I'm fine with these changes.